airflow/executors/kubernetes_executor.py

```diff
@@ -568,15 +601,14 @@ def _create_or_update_secret(secret_name, secret_path):
     def start(self) -> None:
         """Starts the executor"""
         self.log.info('Start Kubernetes executor')
-        self.worker_uuid = KubeWorkerIdentifier.get_or_create_current_kube_worker_uuid()
+        self.worker_uuid = self.job_id
```

Comment:
Lets rename this now please -- it is no longer a uuid.

Could we just use `self.job_id` in place of `self.worker_uuid`?

Reply:
sure